core/drivers/gic.c

```diff
@@ -610,11 +610,11 @@ static void gic_op_set_affinity(struct itr_chip *chip, size_t it,
 #ifdef CFG_DT
 /* Callback for "interrupts" and "interrupts-extended" DT node properties */
 static TEE_Result dt_get_gic_chip_cb(struct dt_pargs *arg, void *priv_data,
-				     struct itr_desc **out_itr_desc)
+				     struct itr_desc **itr_desc_p)
```

Comment:
Why is this a double-pointer? I can't see that it's needed anywhere and it makes the code more complicated.

Reply:
The generic `dt_driver_device_from_xxx()` functions expect a `void **` reference (e.g. [`dt_driver_device_from_node_idx_prop()`](https://github.com/OP-TEE/optee_os/blob/4.0.0/core/include/kernel/dt_driver.h#L156)), hence i used a double pointer reference here also, for consistency. Maybe over killing.

Reply:
I don't quite see what's consistent here, but it does look strange and error-prone to me.

Reply:
Ok, i'll add a fixup patch in the hope I understood your concern.